pkg/helm/client.go

```diff
@@ -125,18 +125,18 @@ func (h CliClient) UpdateRepo(ctx context.Context) error {
 }
 
 // Install installs helm chart with given release name
-func (h CliClient) Install(ctx context.Context, chart, version, release, namespace string, values map[string]string) error {
+func (h CliClient) Install(ctx context.Context, chart, version, release, namespace string, values map[string]string, wait bool) error {
 	log.Debug().Print("Installing helm chart", field.M{"chart": chart, "version": version, "release": release, "namespace": namespace})
 	var setVals string
 	for k, v := range values {
 		setVals += fmt.Sprintf("%s=%s,", k, v)
 	}
 
 	var cmd []string
-	if h.version == V3 {
+	if wait {
 		cmd = []string{"install", release, "--version", version, "--namespace", namespace, chart, "--set", setVals, "--wait", "--create-namespace"}
 	} else {
-		cmd = []string{"install", "--name", release, "--version", version, "--namespace", namespace, chart, "--set", setVals, "--wait"}
+		cmd = []string{"install", release, "--version", version, "--namespace", namespace, chart, "--set", setVals, "--create-namespace"}
```

Comment:
TIOLI: replace the `if/else` with an `if`:

```diff
diff --git a/pkg/helm/client.go b/pkg/helm/client.go
index 4103a26d..159827cf 100644
--- a/pkg/helm/client.go
+++ b/pkg/helm/client.go
@@ -132,11 +132,9 @@ func (h CliClient) Install(ctx context.Context, chart, version, release, namespa
 		setVals += fmt.Sprintf("%s=%s,", k, v)
 	}
 
-	var cmd []string
+	cmd := []string{"install", release, "--version", version, "--namespace", namespace, chart, "--set", setVals, "--create-namespace"}
 	if wait {
-		cmd = []string{"install", release, "--version", version, "--namespace", namespace, chart, "--set", setVals, "--wait", "--create-namespace"}
-	} else {
-		cmd = []string{"install", release, "--version", version, "--namespace", namespace, chart, "--set", setVals, "--create-namespace"}
+		cmd = append(cmd, "--wait")
 	}
 
 	out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
```

Reply:
Looks Good !!